package/MDAnalysis/converters/OpenMM.py

```diff
@@ -125,10 +125,10 @@ def _mda_timestep_from_omm_context(self):
         ts.frame = 0
         ts.data["time"] = state.getTime()._value
         ts.data["potential_energy"] = (
-            state.getPotentialEnergy().in_units_of(u.kilojoule/u.mole)
+            state.getPotentialEnergy().in_units_of(u.kilojoule/u.mole)._value
```

Comment:
Would it be possible to add a quick test for these?

Reply:
Done